src/status_im/domain/market/events.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.domain.market.events
+  (:require
+    [utils.re-frame :as rf]))
+
+(rf/reg-event-fx :domain.market/update-leaderboard
+ (fn [{:keys [db]} [leaderboard]]
+   {:db (update db :domain.market/leaderboard merge leaderboard)}))
+
+(rf/reg-event-fx :domain.market/update-prices
```

Comment:
We have 2 different signals that bring us prices - the one with more details about tokens that comes every 60 seconds and one with only prices update that comes every 15 seconds, so we need to merge prices.